subprojects/docs/src/main/resources/header.html

```diff
@@ -218,8 +218,9 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
         <h3 id="extending-gradle">Extending Gradle</h3>
         <ul>
             <li><a href="https://gradle.org/guides/?q=Plugin%20Development">Plugin Development Tutorials</a></li>
-            <li><a href="custom_tasks.html">Creating Custom Task Types</a></li>
-            <li><a href="custom_plugins.html">Writing Custom Plugins</a></li>
+            <li><a href="custom_tasks.html">Writing Gradle Task Types</a></li>
```

Comment:
Should we use the new chapter headings from the header as well? E.g. `Developing Custom Gradle Task Types` instead of `Writing Gradle Task Types`?

Reply:
I did try that, and they didn't fit very well. We can tweak them later